docs/tutorials/plugin.rst

```diff
@@ -45,7 +45,7 @@ Modifying existing files with patches
 
 We'll start by modifying some of our Open edX settings files. It's a frequent requirement to modify the ``FEATURES`` setting from the LMS or the CMS in edx-platform. In the legacy native installation, this was done by modifying the ``lms.env.yml`` and ``cms.env.yml`` files. Here we'll modify the Python setting files that define the edx-platform configuration. To achieve that we'll make use of two concepts from the Tutor API: :ref:`patches` and :ref:`filters`.
```

Comment:
Somewhere in this tutorial, we ought to link to the new Hooks Catalog landing page you wrote. I also think it would be good to link to the Hooks Catalog from the [Plugins Introduction](https://docs.tutor.overhang.io/plugins/intro.html).

I think it may not immediately click to new plugin devs that the V1 Plugin API is just a CLI around the Hooks API. Adding cross-links to the docs might make that connection more obvious.

Reply:
I agree. I added a few words to the docs page. Let me know if you think this is sufficient.